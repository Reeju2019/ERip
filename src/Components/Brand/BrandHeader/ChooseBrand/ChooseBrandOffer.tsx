import React from 'react'
import './ChooseBrand.css'

const ChooseBrandOffer: React.FunctionComponent = () => {
  return (
    <>
      <div>
        <div className='row'>
          <div className='col-12 offer'>
            <div className='offer-img'>
              <img src='static/Mobile_Services/images/Offer1.png' alt='' />
            </div>

            <div className='offer-img'>
              <img src='static/Mobile_Services/images/Offer2.png' alt='' />
            </div>

            <div className='offer-img'>
              <img src='static/Mobile_Services/images/Offer3.png' alt='' />
            </div>
          </div>
        </div>
      </div>
    </>
  )
}

export default ChooseBrandOffer
