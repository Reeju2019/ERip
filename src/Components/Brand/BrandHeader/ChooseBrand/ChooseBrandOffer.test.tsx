import React from 'react'
import { render, screen } from '@testing-library/react'
import ChooseBrandOffer from './ChooseBrandOffer'
import renderer from 'react-test-renderer'


test('choose brand offer', () => {
  render(<ChooseBrandOffer />)
})

describe('App rendering specification', () => {
    it('SnapShot testing', () => {
      const component = renderer.create(<ChooseBrandOffer />)
      const tree = component.toJSON()
      expect(tree).toMatchSnapshot()
      
    });
})