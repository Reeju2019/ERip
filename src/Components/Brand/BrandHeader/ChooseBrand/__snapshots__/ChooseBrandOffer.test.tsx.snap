// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App rendering specification SnapShot testing 1`] = `
<div>
  <div
    className="row"
  >
    <div
      className="col-12 offer"
    >
      <div
        className="offer-img"
      >
        <img
          alt=""
          src="static/Mobile_Services/images/Offer1.png"
        />
      </div>
      <div
        className="offer-img"
      >
        <img
          alt=""
          src="static/Mobile_Services/images/Offer2.png"
        />
      </div>
      <div
        className="offer-img"
      >
        <img
          alt=""
          src="static/Mobile_Services/images/Offer3.png"
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`App rendering specification SnapShot testing 2`] = `
<div>
  <div
    className="row"
  >
    <div
      className="col-12 offer"
    >
      <div
        className="offer-img"
      >
        <img
          alt=""
          src="static/Mobile_Services/images/Offer1.png"
        />
      </div>
      <div
        className="offer-img"
      >
        <img
          alt=""
          src="static/Mobile_Services/images/Offer2.png"
        />
      </div>
      <div
        className="offer-img"
      >
        <img
          alt=""
          src="static/Mobile_Services/images/Offer3.png"
        />
      </div>
    </div>
  </div>
</div>
`;
